package.json

```diff
@@ -9,6 +9,7 @@
     "pretest": "npm install chromedriver --chromedriver-force-download",
     "test": "npm run test:unit && npm run test:e2e",
     "test:unit": "wct --config-file ./wct.local.conf.json",
+    "test:unit-chrome": "wct -p -l chrome test/unit/*.test.html --npm",
```

Comment:
Niet echt belangrijk, maar de convention is doorgaans "test:unit:chrome"

Reply:
ok